salt-base.dockerfile

```diff
@@ -4,7 +4,7 @@ FROM ubuntu:trusty
 ENV RECLASS_URL https://github.com/tcpcloud/workshop-salt-model.git
 ENV RECLASS_BRANCH docker
 ENV REPO_URL "http://apt.tcpcloud.eu/nightly/"
-ENV REPO_COMPONENTS "main security extra tcp tcp-salt"
+ENV REPO_COMPONENTS "main security extra tcp tcp-salt kilo"
```

Comment:
We can instead setup openstack repo during linux state run in per-formula dockerfile.